### PR TITLE
apply Proofreading suggestions to c14

### PR DESCRIPTION
**Pull request by jannes-m**

### CODE REVIEW

**Comment by jannes-m** [comment]
https://github.com/geocompx/geocompr/blob/63d10cbd2b54822ba022d1810a71f3d8e82e51b0/14-location.Rmd#L135

This is displayed in the pdf output as:  Figure reﬀig:census-stack 

I thought we had this long overcome, any ideas, @Nowosad?

**Comment by Nowosad** [comment]
@jannes-m I think that this solution worked at some point. Can you avoid using refs in the captions somehow?

**Comment by jannes-m** [comment]
Yes, I can 😊

File: _14-ex.Rmd [hunk 1]
```diff
@@ -13,8 +13,8 @@ library(spDataLarge)
 
 E1. Download the csv file containing inhabitant information for a 100 m cell resolution (https://www.zensus2011.de/SharedDocs/Downloads/DE/Pressemitteilung/DemografischeGrunddaten/csv_Bevoelkerung_100m_Gitter.zip?__blob=publicationFile&v=3).
 Please note that the unzipped file has a size of 1.23 GB.
-To read it into R you can use `readr::read_csv`.
-This takes 30 seconds on a machine with 16 GB RAM.
+To read it into R, you can use `readr::read_csv`.
+This takes 30 seconds on a machine with 16-GB RAM.
```
Comment (Robinlovelace) [inline _14-ex.Rmd:17]: 16-GB looks fine, not sure if we're doing that throughout. If not I tentatively suggest not making that change. Minor niggle.

**Review by Robinlovelace — APPROVED** [review]

File: _14-ex.Rmd (hunk 1 above)
Comment (jannes-m) [reply _14-ex.Rmd:17]: you are right, I'll change back

**Pull request merged**